src/init.cpp

```diff
@@ -788,7 +788,9 @@ bool AppInit2(boost::thread_group& threadGroup)
     LogPrintf("Using at most %i connections (%i file descriptors available)\n", nMaxConnections, nFD);
     std::ostringstream strErrors;
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
+    // nScriptCheckThreads == 0 means we want to display it as 1 thread
+    unsigned int logScriptCheckThreads = (nScriptCheckThreads == 0) ? 1 : nScriptCheckThreads;
+    LogPrintf("Using %u threads for script verification\n", logScriptCheckThreads);
```

Comment:
Damn right, my initial thoughts were that they HAVE to be checked anyway, so 0 threads makes no sense. But without threads just means we do it directly in the function and not in separate threads right?